chat-plugins/wisp.js

```diff
@@ -1095,6 +1095,50 @@ exports.commands = {
 		"/background set [user], [image] - Sets a users profile background.",
 		"/background delete [user] - Deletes a users profile background.",
 	],
+	music: {
+		set: function (target, room, user) {
+			if (!this.can('music')) return false;
```

Comment:
I'm a bit confused as to what is required, especially considering backgrounds, title, etc don't have any lines in config-example.js as far as I could see.